README.md

```diff
@@ -43,54 +42,27 @@ Contents:
 
 <!-- TOC -->
 
+- [How to build Valora](#build)
 - [Celo's Mission - Prosperity for All](#mission)
-- [The Celo Stack](#stack)
+- [The Valora Stack](#stack)
 - [Documentation](#docs)
 - [Issues](#issues)
 - [Repo Structure](#repo)
 - [Contributing](#contributing)
 - [Ask Questions, Find Answers, Get in Touch](#ask)
 - [License](#license)
   <!-- /TOC -->
+  
+## 🥅 <a id="build"></a>How to Build Valora Locally 
+Follow instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md#running-the-mobile-wallet). (This part of the READ.me is in construction! To be rewritten) 
```

Comment:
We should move those setup instructions out of the monorepo to here.

Reply:
👍

Reply:
the instructions are actually here: https://github.com/celo-org/wallet/tree/main/packages/mobile will link to it instead.